stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -764,6 +861,439 @@ fn block_descendant() {
     );
 }
 
+#[test]
+fn block_info_primary_testnet() {
+    block_info_tests(true)
+}
+
+#[test]
+fn block_info_other_testnet() {
+    block_info_tests(false)
+}
+
+fn block_info_tests(use_primary_testnet: bool) {
```

Comment:
I'm curious why this test doesn't consider Clarity 2? That's the currently-active Clarity.

Reply:
Because the code under test is identical in both Clarity 2 and Clarity 1 (it's not just identical code, its the same exact codepath). Testing both just adds bloat to the test.